packages/ai/src/generate-text/generate-text.ts

```diff
@@ -606,8 +606,8 @@ async function executeTools<TOOLS extends ToolSet>({
                 selectTelemetryAttributes({
                   telemetry,
                   attributes: {
-                    'ai.toolCall.result': {
-                      output: () => JSON.stringify(result),
+                    'ai.toolCall.output': {
+                      output: () => JSON.stringify(output),
```

Comment:
we are not changing telemetry attributes for backawards compat. telemetry will be reworked in 5.1

Reply:
yes only toolCall.input and toolCall.output afaik. worth comparing to v4